.env.example

```diff
@@ -29,4 +29,6 @@ REDIS_PASSWORD=password
 
 STRIPE_KEY=test
 
-CI=false
\ No newline at end of file
+MAILGUN_API_KEY=test
+MAILGUN_DOMAIN=test
+MAILGUN_SENDER=no-reply@resonate.coop
```

Comment:
Still need to get the actual MAILGUN keys for our servers. Locally I'm just using my own mailgun sandbox keys.